tests/data/power_flow/asym-line/test.dss

```diff
@@ -0,0 +1,22 @@
+Clear
+
+Set DefaultBaseFrequency=50
+
+! Swing or Source Bar
+New circuit.test_1 phases=3 pu=1.0 basekv=10 bus1=Node1.1.2.3.4
+
+! LineCodes
+New LineCode.1428_VMvKh_4x50mm2_Al___4x2p5mm2_Cu nphases=4 C1=0.54 C0=0.32 Normamps=115.0 Units=km
+~ Rmatrix = [0.6904 |0.0495 0.6904 |0.0492 0.0495 0.6904 | 0.0495 0.0492 0.0495 0.6904]
+~ Xmatrix = [0.8316 |0.7559 0.8316 |0.7339 0.7559 0.8316 | 0.7559 0.7339 0.7559 0.8316]
+
+! Lines 
+New Line.Cable692 Phases=4 Bus1=Node1.1.2.3.4 Bus2=Node2.1.2.3.4 LineCode=1428_VMvKh_4x50mm2_Al___4x2p5mm2_Cu Length=234.79 Units=m 
+New Line.Cable551 Phases=4 Bus1=Node2.1.2.3.4 Bus2=Node3.1.2.3.4 LineCode=1428_VMvKh_4x50mm2_Al___4x2p5mm2_Cu Length=32.914 Units=m 
+
+! Load Definitions 
+New Load.Home853_1_Ph1 Bus1=Node1.1.2.3.4  Phases=1 Conn=wye Model=1 kV=0.230 kW=1 kvar=1.0 Vmaxpu=1.5 Vminpu=0.60 
```

Comment:
```suggestion
New Load.Home853_1_Ph1 Bus1=Node2.1  Phases=1 Conn=wye Model=1 kV=5.78034 kW=1 kvar=1.0
New Load.Home853_1_Ph2 Bus1=Node2.2 Phases=1 Conn=wye Model=1 kV=5.78034 kW=2 kvar=2.0
New Load.Home853_1_Ph3 Bus1=Node2.3 Phases=1 Conn=wye Model=1 kV=5.78034 kW=3 kvar=3.0
```
@leovsch  Also, Individual phases can be put in this way. This creates an asym load with 1,2,3 as p and q specified in PGM.